contracts/helpers/DigestHelper.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.13;
+// prettier-ignore
```

Comment:
thanks for updating to the new typescript test file! looks great.

two comments here, can you change all the pragma solidity versions to carat `^` instead of `>=` (this has been recently done on main)

Also you won't need this `prettier-ignore` with latest main now having prettier multi-line import support

Reply:
Awesome :D Thanks! Just implemented these changes.

Reply:
If you merge in changes from main it was fixed in a recent PR

Reply:
And looks great! Maybe re-add a newline in this file below the pragma version since it's hugging the first import now.

Reply:
Got that working! Merged main and pushed some linter changes from `lint:fix`. Also added that new line :D